server/handlers/clients.js

```diff
@@ -13,14 +13,11 @@ const {
   getConfig
 } = configHelpers;
 
-let configsMap = null;
-
 function getClientInfo(req, res) {
-  const { logger, config } = req;
-  const clientConfigs = loadClientConfigs(config);
+  const { logger, clients } = req;
   const clientInfo = {};
 
-  for (let client of clientConfigs) {
```

Comment:
It may not be useful here, but `continue` and `break` work with `for ... of` but don't with `forEach`